lib/stoplight/configuration.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+
+module Stoplight
+  # An immutable +Stoplight::Light+ configuration object that could be
```

Comment:
I think that it would be better if we mention that we return the `Stoplight::Configuration` object here.